packages/intent-filler/src/config/chain.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Enum representing different chains.
+ */
+export enum Chains {
```

Comment:
We should probably autogenerate the contents of this file from a toml config file.

Reply:
I have added a toml that is used for config generation, let me know if it resolves your comment. Otherwise please share the messaging relayer repo, so that I can see the example. Sorry, saw your edited msg now.